<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat BOT Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>64</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ログインID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>64</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>パスワード</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginPushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ログイン</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="replyTextEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="requestTextEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPushButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>610</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>送信
(Ctrl+Enter)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historyPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>履歴取得</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接続先</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remoteLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>ws://127.0.0.1:55554</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>接続</string>
    </property>
   </widget>
   <widget class="QListWidget" name="historyListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>ダブルクリックで実行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newChatPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>新しいチャット</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>remoteLineEdit</tabstop>
  <tabstop>connPushButton</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>passLineEdit</tabstop>
  <tabstop>loginPushButton</tabstop>
  <tabstop>newChatPushButton</tabstop>
  <tabstop>historyListWidget</tabstop>
  <tabstop>historyPushButton</tabstop>
  <tabstop>replyTextEdit</tabstop>
  <tabstop>requestTextEdit</tabstop>
  <tabstop>sendPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
